addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11636,13 +11636,13 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#21472"
-msgid "Include All Seasons and Specials"
+msgid "Include all seasons and specials"
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Include All Seasons and Specials" with label #21472
 #: system/settings/settings.xml
 msgctxt "#21473"
-msgid "Whether or not to consider All Seasons and Special items in the unwatched item selection."
+msgid "Whether or not to consider all seasons and special items in the unwatched item selection."
```

Comment:
So you are saying it should be


uNiversaI commented 2 days ago

Similar to #7054 but on resource.language.en_gb

@MartijnKaijser @jjd-uk @Jalle19

Second commit could be squashed, was done separately as it contained the typos / misspeling
and other non capitalization fixes.

May have missed one or other instances (eyes gone after a while).

Please indicate which lines to fix on review comments.
@Jalle19 if you feel like parting with your excellent review skills.

unused strings were considered but I think it needs some deeper research so left them out at this time.
@uNiversaI uNiversaI changed the title from [strings] capitalized second/third word to [strings] Take 2 - fix capitalized second/third word 2 days ago
@uNiversaI uNiversaI changed the title from [strings] Take 2 - fix capitalized second/third word to [strings] take 2 - fix capitalized second/third word 2 days ago
@uNiversaI uNiversaI referenced this pull request 2 days ago
Closed
[strings] take 3 - fix misspellings/typos, apostrophes/hyphens and cosmetics #7081
@uNiversaI uNiversaI changed the title from [strings] take 2 - fix capitalized second/third word to [strings] take 2 - fix capitalized second/third word + fix typos/misspelling and cleanup. 2 days ago
@MartijnKaijser
Show outdated diff
MartijnKaijser commented on an outdated diff a day ago
@da-anda da-anda commented on the diff a day ago
View full changes
addons/resource.language.en_gb/resources/strings.po
@@ -8646,7 +8646,7 @@ msgid "The PVR manager has been enabled without any"
 msgstr ""
 
 msgctxt "#19242"
-msgid "enabled PVR Add-on. Enable at least one Add-on"
+msgid "enabled PVR add-on. Enable at least one add-on"
@da-anda Collaborator
da-anda added a note a day ago

While you're at it - uppercase first letter maybe?

edit: ah sorry, seems to be part of a concatenated string - wasn't obvious in GIT diff on first sight
@MartijnKaijser Owner
MartijnKaijser added a note a day ago

we need to clean up those concatenated strings as we now have a textbox for dialogs instead of separate lines. Something for later on
@da-anda Collaborator
da-anda added a note a day ago

absolutely
@da-anda
Show outdated diff
da-anda commented on an outdated diff a day ago
uNiversaI added some commits 2 days ago
		[strings] capitalized second/third word … 	527f09d
		[strings] typos/misspellings/cleanup and cosmetics. 	a2f6408
@da-anda da-anda commented on the diff an hour ago
View full changes
addons/resource.language.en_gb/resources/strings.po
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
@da-anda Collaborator
da-anda added a note an hour ago

why going the other way around here (lower- to uppercase)?
@uNiversaI
uNiversaI added a note an hour ago

It is - World Wide Web in any reputable dictionary.
http://en.wikipedia.org/wiki/World_Wide_Web
http://www.merriam-webster.com/dictionary/world%20wide%20web

So anything with Web interface, WebServer Webserver Web page etc etc is correct.

I changed webserver -> Webserver also throughout in this file
@da-anda da-anda commented on the diff an hour ago
View full changes
addons/resource.language.en_gb/resources/strings.po
((6 lines not shown))
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Include All Seasons and Specials" with label #21472
 #: system/settings/settings.xml
 msgctxt "#21473"
-msgid "Whether or not to consider All Seasons and Special items in the unwatched item selection."
```
+msgid "Whether or not to consider \"All seasons\" and "\Special\" items in the unwatched item selection."
```